### ISSUE TITLE
[Verification/Geofencing]: Decrease the minimum of a Circle radius

### ISSUE THREAD

**Issue by maxl2287** [camaraproject/DeviceLocation#282]
**Problem description**
As of now the minimum of a radius is set to `2000` meters, which is quite too large as a minimum.
We discovered the issue as we wanted to test a geofencing creation inside of a building, but we were forced to use a geofence with a minimum of 2 kilometers.

**Possible evolution**
Decrease the minimum to a smaller value, e.g. 50 meters

**Comment by alpaycetin74** [comment]
This was probably a result of privacy concerns. The minimum in the API specification could be even much smaller, but the implementer must be made aware of legal implications and define a minimum for their use case.

**Comment by jlurien** [comment]
I agree, that we need to indicate that implementations may consider a minimum radius as valid input, due to privacy or implementation constraints, and as @bigludo7 proposes, define a specific error to signal that the input area is not supported,

**Comment by javier-carrocalabor** [comment]
I have checked this with our legal team and it's ok for me to support this.
